R/process_extract_delayed_discharges.R

```diff
@@ -22,91 +22,96 @@ process_extract_delayed_discharges <- function(data,
   # Check that the supplied year is in the correct format
   year <- check_year_format(year)
 
-  # Data Cleaning---------------------------------------
+  # Specify years available for running
+  if (year >= 1617) {
```

Comment:
Again a simplification is probably something like 

```suggestion
  # Specify years available for running
  if (year < 1617) {
return(NULL)
}

... rest of code.
```

Reply:
Thanks James, I agree, those look much easier to read!